modules/serverless/pages/serving/serving-transport-encryption.adoc

```diff
@@ -0,0 +1,446 @@
+
+==== {ServerlessProductName} Serving system-internal encryption
+
+image::serving-encryption/encryption-system-internal.drawio.svg[]
+
+This part enables transport encryption for system internal components (Ingress-Gateway, Activator, Queue-Proxy). These components are hosting TLS endpoints when this configuration is used:
+
+* To get the certificates, {ServerlessProductName} relies on cert-manager. cert-manager needs to be installed and configured for the feature to work.
+* Specific SANs are used to verify each connection. Each component needs to trust the CA that signed the certificates. For this, {ServerlessProductName} system components will consume and trust a bundle of CAs. The CA bundle needs to be provided by the cluster administrator.
+
+
+== Identify and select an appropriate certificate issuer [[issuer_selection]]
+
+[INFO]
+====
+Issuers here are referring to cert-manager `Issuers` and `ClusterIssuers`. They represent certificate authorities (CAs) that can generate signed certificates by honoring certificate signing requests.
+Reference: https://cert-manager.io/docs/concepts/issuer/
+====
+
+In general, you can refer to the cert-manager documentation (https://cert-manager.io/docs/configuration/issuers/). There are examples available for:
+
+* A custom CA stored in a K8s secret
+* HTTP-01 challenges, e.g. Let's encrypt
+* DNS-01 challenges
+* Self-signed issuers
+
+[IMPORTANT]
+====
+Please note, that not all issuer types work for each Knative Serving encryption feature. See below for details.
+====
+
+Depending on the feature you would like to use, {serverlessproductname} requires your certificate issuer to be capable of signing the following certificates:
+
+For the `cluster-local-domain-tls` feature, the issuer needs to be able to sign certificates for cluster-local domains like:
+
+* `myapp.<namespace>`
+* `myapp.<namespace>.svc`
+* `myapp.<namespace>.svc.cluster.local`
+
+As the CA usually is not within the cluster, verification via ACME protocol (DNS01/HTTP01) is not possible. You can use an issuer that allows creating these certificates (e.g. cert-manager's CA issuer).
+
+For the `system-internal-tls` feature, the CA must be able to sign certificates with a subject alternative name (SAN) of:
+
+* `kn-routing`
+* `kn-user-<namespace>` (<namespace> is each namespace where Knative Services are/will be created)
+* `data-plane.knative.dev`
+
+These SANs are needed for Knative to verify connections between the internal components.
+As this is also not possible via ACME protocol (DNS01/HTTP01), you need to configure an issuer that allows creating these certificates (e.g. cert-manager's CA issuer).
+
+
+== Installing and configuring {ServerlessProductName} transport encryption
+
+=== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the {certmanageroperatorname}.
```

Comment:
This should be mentioned above when talking about that CertManager needs to be installed (see my previous comments)